kubernetes/akka-cluster.yml

```diff
@@ -18,15 +18,22 @@ spec:
     spec:
       containers:
       - name: appka
-        image: akka-sample-cluster-kubernetes-scala:0.1
+        image: akka-sample-cluster-kubernetes-scala:latest
         imagePullPolicy: Never
         readinessProbe:
           httpGet:
             path: /ready
-            port: 8558
+            port: management
+          periodSeconds: 10
+          failureThreshold: 3
+          initialDelaySeconds: 10
         livenessProbe:
-          tcpSocket:
-            port: 8558
+          httpGet:
+            path: "/alive"
+            port: management
+          periodSeconds: 10
+          failureThreshold: 5
+          initialDelaySeconds: 20
```

Comment:
in the guide it is: https://developer.lightbend.com/guides/openshift-deployment/lagom/forming-a-cluster.html#health-checks